lib/container/ModuleFederationPlugin.js

```diff
@@ -65,7 +65,25 @@ class ModuleFederationPlugin {
 					library,
 					filename: options.filename,
 					runtime: options.runtime,
-					exposes: options.exposes
+					exposes: options.exposes,
+					remotes: Object.keys(options.remotes || {})
+				}).apply(compiler);
+				const sharedExposes = Object.keys(options.exposes).reduce(
+					(items, exposeKey) => ({
+						...items,
+						[options.exposes[exposeKey]]: {
+							singleton: true,
```

Comment:
We cannot assume configurations here.

Reply:
hosts that are exposing modules needs to be included as shared also because they needs to be available to child MF